src/components/App/App.css

```diff
@@ -0,0 +1,23 @@
+/* App.css */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  background-color: #f5f5f5;
+}
+
+.app {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+}
+
+.app main {
```

Comment:
`.app main {
`

According to BEM, it's not allowed to use **tags** in `css` for styling. You need to add **class names** for the elements